src/controllers/movie.js

```diff
@@ -0,0 +1,84 @@
+const { Prisma } = require("@prisma/client")
+const prisma = require('../utils/prisma')
+
+
+const getAllMovies = async (req, res) => {
+
+    const movies = await prisma.movie.findMany({
+        include: {
+            screenings: true,
+          }
+    })
+    return res.status(200).json({ movies })
```

Comment:
.status(200) doesn't have to be included as it defaults to 200, but it's nice to see you did it anyways as it makes it clearer to see what's going on